introduction.md

```diff
@@ -1,2 +1,6 @@
 ey this is alexis. My favorite path so far is git!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?